tests/ui/layout/aggregate-lang/struct-align.rs

```diff
@@ -0,0 +1,31 @@
+//@ run-pass
+//@ reference: layout.aggregate.struct-size-align
+//@ edition: 2018
+
+#[repr(align(64))]
+#[derive(Copy, Clone)]
+#[allow(dead_code)]
+pub struct Overaligned(u8);
+
+#[allow(dead_code)]
+struct ReprRustStruct {
+    x: i32,
+    y: [u32; 4],
+    z: f32,
+    a: u128,
+    b: Overaligned,
+}
+
+#[cfg_attr(test, test)]
```

Comment:
Question: why is this needed?

Reply:
This was copied out of the tests I wrote for https://github.com/lccc-project/rust-spec-conformance, which uses `--test` for run-pass tests.

Reply:
Ah okay. Please remove these `--test` annotations, then we can merge these tests.

Reply:
You can r=me after PR CI is green and removing those `--test`-related annotations.